CONTRIBUTING.md

```diff
@@ -293,37 +287,3 @@ https://github.com/SeleniumHQ/selenium/wiki
 https://groups.google.com/forum/#!forum/selenium-users
 
 https://github.com/SeleniumHQ/selenium/wiki/Selenium-Help
-
```

Comment:
This needs to stay. People still sometimes use it, and we refer to this section of the docs.